packages/stream_chat_flutter/lib/src/message_input/stream_message_send_button.dart

```diff
@@ -62,19 +62,25 @@ class StreamMessageSendButton extends StatelessWidget {
       );
     }
 
+    final idleSendIcon = switch (idleSendButton) {
+      final idleButton? => idleButton,
+      _ => const StreamSvgIcon(icon: StreamSvgIcons.sendMessage),
+    };
+
+    final activeSendIcon = switch (activeSendButton) {
+      final activeButton? => activeButton,
+      _ => const StreamSvgIcon(icon: StreamSvgIcons.circleUp),
+    };
+
     final theme = StreamMessageInputTheme.of(context);
+    final icon = isIdle ? idleSendIcon : activeSendIcon;
     final onPressed = isIdle ? null : onSendMessage;
     return StreamMessageInputIconButton(
       key: const Key('send_button'),
-      icon: StreamSvgIcon(icon: _sendButtonIcon),
+      icon: icon,
       color: theme.sendButtonColor,
       disabledColor: theme.sendButtonIdleColor,
       onPressed: onPressed,
     );
   }
```

Comment:
Here you wrap the `activeButton` into a `StreamMessageInputIconButton`, which doesn't make sense to me.

 Probably needs something like this:
```dart
    final effectiveActiveSendButton = switch (activeSendButton) {
      final activeButton? => activeButton,
      _ => StreamMessageInputIconButton(
            key: const Key('send_button'),
            icon: StreamSvgIcon(icon: _sendButtonIcon),
            icon: const StreamSvgIcon(icon: StreamSvgIcons.circleUp),
            color: theme.sendButtonColor,
            disabledColor: theme.sendButtonIdleColor,
            onPressed: onPressed,
          );
    };
```